beginners/medium/001.reverse-a-number/206.saadman-galib.reverse-a-number.py

```diff
@@ -0,0 +1 @@
+print(input()[::-1])
```

Comment:
create a video by explaining this line and send to to strangers channel of slack

Reply:
Ok bhaiya